explorer_frontend/src/features/healthcheck/init.ts

```diff
@@ -0,0 +1,30 @@
+import { sample } from "effector";
+import { interval } from "patronum";
+import { $endpoint } from "../account-connector/model";
+import { loadedPlaygroundPage } from "../code/model";
+import { $isPageVisible, $rpcIsHealthy, checkRpcHealthFx, pageVisibilityChanged } from "./model";
+
+$isPageVisible.on(pageVisibilityChanged, (_, isVisible) => isVisible);
+document.addEventListener("visibilitychange", () => {
```

Comment:
it's better to be in react useEffect because that's ok for production but will be a million of event listener in case of dev mode

Reply:
Do you try to update this file in this case? It's usually do not delete previous listener

Reply:
Yep, checked again with this file and with changes in UI - no listeners overflow